integrationtests/Paket.IntegrationTests/UpdatePackageSpecs.fs

```diff
@@ -141,18 +141,6 @@ let ``#1703 resolves locally``() =
     update "i001703-local" |> ignore
 
 
-[<Test>]
-let ``#1635 should tell about auth issue``() =
```

Comment:
Are you sure this is not a regression?

Reply:
yes. because I looked at the error message.

Reply:
yeah, you were right. it was a regression. fixed